src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -994,6 +994,7 @@ ss::future<cluster::init_tm_tx_reply> tx_gateway_frontend::init_tm_tx(
               txlog.trace,
               "[tx_id: {}] waiting for {} topic to apper in metadata cache, "
               "retries left: {}",
+              tx_id,
```

Comment:
I think I just fixed this one: https://github.com/redpanda-data/redpanda/pull/15187

Reply:
Great I'll remove it

Reply:
Mmm wonder why it wasn't flagged as merge conflict

Reply:
I was wondering that, feel free to merge and we'll see what happens? 😄

Reply:
here goes 🤞